fsthttp/request.go

```diff
@@ -115,6 +116,8 @@ type Request struct {
 	// RequestID is the current Fastly request ID
 	RequestID string
 
+	ImageOptimizerOpts *imageopto.Opts
```

Comment:
Do you think it may make sense to name this `ImageOptimizerOptions`? I ask because this struct already has `CacheOptions` and `DecompressResponseOptions`.

The Rust SDK uses `ImageOptimizerOptions`, and that was also what it was named in the rough design doc we shared.

Reply:
Yeah, good point.  I'll handle this.

Reply:
Sorry, took a while to get to this: https://github.com/fastly/compute-sdk-go/pull/208